Not found. Configuration page

The installation of the corresponding version started successfully，But there's another problem，After successful startup：
Where is the nginx configuration page???

![微信截图_20221117095036](https://user-images.githubusercontent.com/54729175/202334362-d144a9a5-fcbb-4f72-8604-cb12ee380411.png)


![微信截图_20221117094302](https://user-images.githubusercontent.com/54729175/202333451-94f7549a-e39d-48f8-8350-ec8b5c7742f5.png)

_Originally posted by @ZhangWeike2000 in https://github.com/digitalocean/nginxconfig.io/issues/407#issuecomment-1317939329_

I got error:
`Uncaught (in promise) Error: Cannot find module './de/languages.js'
    at app.js:2083:11
    at async getI18n (setup.js?bfd8:48:1)`
Maybe webpack failed to pack language.js, my environment is:
`
$ node -v
v18.12.1

$ npm -v
8.19.2

$ npm info webpack
webpack@5.75.0 | MIT | deps: 24 | versions: 803
`

--- a/src/nginxconfig/i18n/setup.js
+++ b/src/nginxconfig/i18n/setup.js
@@ -46,7 +46,7 @@ export const getI18n = async () => {
         if (availablePack === defaultPack) continue;
         if (i18nPacks[availablePack]) continue;
         const { default: languageData } = await import(
-            /* webpackInclude: /i18n\/[^/]+\/languages\.js$/ */
+            /* webpackInclude: /i18n[\/\\][^\/\\]+[\/\\]languages\.js$/ */
             /* webpackMode: "eager" */
             `./${toSep(availablePack, '-')}/languages.js`
         );
@@ -72,7 +72,7 @@ const loadLanguagePack = async pack => {
     // Load in the full pack
     // Use webpack magic to only build chunks for lang/index.js
     const { default: packData } = await import(
-        /* webpackInclude: /i18n\/[^/]+\/index\.js$/ */
+        /* webpackInclude: /i18n[\/\\][^\/\\]+[\/\\]index\.js$/ */
         /* webpackMode: "lazy" */
         `./${toSep(pack, '-')}/index.js`
     );

